src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1566,6 +1567,22 @@ public String toVariableNameString(EnchantmentOffer eo) {
 					"Individual attributes of a snapshot cannot be modified or retrieved.")
 				.requiredPlugins("Minecraft 1.20.2+")
 				.since("INSERT VERSION")
+				.parser(new Parser<>() {
+					@Override
+					public boolean canParse(ParseContext context) {
+						return false;
+					}
+
+					@Override
+					public String toString(EntitySnapshot snapshot, int flags) {
+						return EntityUtils.toSkriptEntityData(snapshot.getEntityType()).toString() + " snapshot";
+					}
+
+					@Override
+					public String toVariableNameString(EntitySnapshot snapshot) {
+						return EntityUtils.toSkriptEntityData(snapshot.getEntityType()).toString() + " snapshot";
```

Comment:
```suggestion
						return toString(snapshot, 0);
```
to avoid some duplication